frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -55,6 +55,15 @@ SPDX-License-Identifier: Apache-2.0
         </v-btn>
       </v-col>
     </v-row>
+    <v-row key="networkWarning" class="list-item" ref="networkWarningRef">
+      <v-alert type="warning" tile prominent v-if="hasAdditionalZonesNetworkConfiguration" class="d-flex flex-grow-1">
+        Adding addtional zones will extend the zone network configuration by adding new networks to your cluster:
+        <pre>
+<code>{{additionalZonesNetworkConfiguration}}</code>
```

Comment:
<img width="993" alt="Screenshot 2022-05-03 at 19 13 09" src="https://user-images.githubusercontent.com/5526658/166504598-1f76f2b9-3f32-45e5-88c3-6c305fbefaf0.png">
I'd remove the spacing between the lines

Reply:
done

Reply:
use code block

Reply:
use code block

Reply:
do not scroll

Reply:
do not scroll